testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -388,7 +388,10 @@ public void testMaterializedView()
         String catalogName = getSession().getCatalog().orElseThrow();
         String schemaName = getSession().getSchema().orElseThrow();
         String viewName = "test_materialized_view_" + randomTableSuffix();
-        assertUpdate("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM nation");
+        assertUpdate(
+                "CREATE MATERIALIZED VIEW " + viewName + " " +
+                        "AS SELECT nationkey \"nationkey 0\", name \"name 1\", regionkey \"regionkey 2\", comment \"comment 3\" " +
```

Comment:
Maybe smoke test should remain simple ? We are covering this in BCT too, so not sure why we want to test it here as well